src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,20 @@
 package core.basesyntax;
 
+import core.basesyntax.model.Bulldozer;
+import core.basesyntax.model.Excavator;
+import core.basesyntax.model.Machine;
+import core.basesyntax.model.Truck;
+
 public class MainApp {
+    private Machine truck = new Truck();
+    private Machine bulldozer = new Bulldozer();
+    private Machine excavator = new Excavator();
+    private Machine[] machines = new Machine[] {truck, bulldozer, excavator};
```

Comment:
Do not declare this variables at the class-level, because you can explicitly create them in the array initialization. also this array should be local variable.